test/k8s/bgp.go

```diff
@@ -66,15 +66,14 @@ var _ = SkipDescribeIf(
 			BeforeAll(func() {
 				frr = applyFRRTemplate(kubectl, ni)
 				kubectl.ApplyDefault(frr).ExpectSuccess("Unable to apply rendered template %s", frr)
+				kubectl.WaitForSinglePod("kube-system", "frr", 30*time.Second)
```

Comment:
You ignore the error check here because the next line will give us the guarantee, right?

Reply:
No, I just forgot to handle it 😅 Fixed in the latest push.